fuzzers/005-tilegrid/cfg_int/Makefile

```diff
@@ -1,3 +1,3 @@
-N ?= 4
+N ?= 10
```

Comment:
This was causing some fuzzer instability.  It might be the cause of https://github.com/SymbiFlow/prjxray/pull/1227?notification_referrer_id=MDE4Ok5vdGlmaWNhdGlvblRocmVhZDcxMDIxNDIxODo1MzcwNzQ%3D&notifications_query=is%3Aunread#issuecomment-587601068